minor langunage update in home page and upgraded minor version (#11)
NREL-Distribution-Suites/grid-data-models: README.md

# Grid Data Models (GDM)

> GDM is a python package containing [pydantic](https://docs.pydantic.dev/latest/) for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/) and intended to be open sourced in the future.

Reviewer: This sentence seems awkward. Are you intending something like this?
```suggestion
GDM is a python package containing data models for power system assets and datasets. This package is actively being developed at [National Renewable Energy Laboratory (NREL)](https://www.nrel.gov/).
```
Note that I removed the open source language.

Author: Thanks for the edit.

## Installation

You can install latest version of `grid-data-models` from PyPi.

```bash
pip install grid-data-models
```

## Why Grid Data Models ?

In an effort to reduce code duplication and provide client packages a standard interface to interact with power system data, a group of 
research engineers at NREL is working on developing standard data models. Features:

> - **Builtin validation layer:** Use of [pydantic](https://docs.pydantic.dev/latest/) allows us to validate model fields.

Reviewer: I would use `built-in`.

Author: I like that.

- **Time series data management:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) package which enables [efficient time series data management](https://nrel.github.io/infrasys/explanation/time_series.html) by sharing arrays across components and offloading system memory. For example, we can attach time series power consumption data to a load profile.
- **Builtin unit conversion:** GDM leverages [pint](https://pint.readthedocs.io/en/stable/) for unit conversion for power system quantities. For e.g power, voltage, time etc.

> - **JSON serialization/deserializatin:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) to serialize and deserialize distribution system containing power system components and time series data attached to components.

Reviewer: Typo: deserialization

> - **JSON serialization/deserializatin:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) to serialize and deserialize distribution system containing power system components and time series data attached to components.

Reviewer: ```
JSON serialization/deserialization:** GDM uses [infrasys](https://github.nrel.gov/CADET/infrastructure_systems) to serialize and deserialize distribution system components to/from JSON.
```
Note that we don't serialize time series data to JSON.

Author: Got it.
